src/util/address.rs

```diff
@@ -242,27 +240,14 @@ impl FromStr for WitnessVersion {
 }
 
 impl WitnessVersion {
-    /// Converts 5-bit unsigned integer value matching single symbol from Bech32(m) address encoding
-    /// ([`bech32::u5`]) into [`WitnessVersion`] variant.
-    ///
-    /// # Returns
-    /// Version of the Witness program
-    ///
-    /// # Errors
-    /// If the integer does not corresponds to any witness version, errors with
-    /// [`Error::InvalidWitnessVersion`]
+    /// Convert a 5-bit unsigned integer value matching single symbol from
+    /// Bech32(m) address encoding ([`bech32::u5`]) into [`WitnessVersion`]
+    /// variant.
     pub fn from_u5(value: ::bech32::u5) -> Result<Self, Error> {
         WitnessVersion::from_num(value.to_u8())
     }
 
-    /// Converts 8-bit unsigned integer value into [`WitnessVersion`] variant.
-    ///
-    /// # Returns
-    /// Version of the Witness program
-    ///
-    /// # Errors
-    /// If the integer does not corresponds to any witness version, errors with
-    /// [`Error::InvalidWitnessVersion`]
+    /// Convert an 8-bit unsigned integer value into [`WitnessVersion`] variant.
```

Comment:
I think Errors section is still useful

Reply:
No worries, if they are useful to you I'll put them back in. Thanks.

Reply:
Added these back in and fixed the return variant to mirror the code.